File: AdminUi/test/AdminUi.Tests.Integration/Models/CreateClientResponse.cs
namespace Backbone.AdminUi.Tests.Integration.Models;
public class CreateClientResponse
{
    public string ClientId { get; set; }
    public string DisplayName { get; set; }
    public string ClientSecret { get; set; }
    public DateTime CreatedAt { get; set; }
    public int? MaxIdentities { get; set; }
}

File: Applications/AdminUi/packages/admin_api_types/lib/src/clients/create_client_response.dart
import 'package:json_annotation/json_annotation.dart';

part 'create_client_response.g.dart';

@JsonSerializable()
class CreateClientResponse {
  final String clientId;
  final String displayName;
  final String clientSecret;
  final String defaultTier;
  final DateTime createdAt;
  final int? maxIdentities;

  CreateClientResponse({
    required this.clientId,
    required this.displayName,
    required this.clientSecret,
    required this.defaultTier,
    required this.createdAt,
    required this.maxIdentities,
  });

  factory CreateClientResponse.fromJson(dynamic json) => _$CreateClientResponseFromJson(json as Map<String, dynamic>);
  Map<String, dynamic> toJson() => _$CreateClientResponseToJson(this);
}
